SEACAS: Bad header path in Iovs_exodus_DatabaseIO.h

## Bug Report
@trilinos/seacas

### Description
PR #8951 breaks https://github.com/exawind/nalu-wind with the following error

```
In file included from /home/sanantha/exawind/source/nalu-wind/src/Realm.C:158:
/nopt/nrel/ecom/exawind/exawind-2020-09-21/install/gcc/trilinos-2021-04-04/include/Iovs_exodus_DatabaseIO.h:18:10: fatal error: visualization/utils/Iovs_Utils.h: No such file or directory
 #include <visualization/utils/Iovs_Utils.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

This line https://github.com/trilinos/Trilinos/blob/7ecf9753b5c093035f3d877bf6ccdbc2a6459939/packages/seacas/libraries/ioss/src/visualization/exodus/Iovs_exodus_DatabaseIO.h#L18 should be 

```C++
#include <Iovs_Utils.h>
``` 

when Trilinos is used from an installed location. 

### Steps to Reproduce
1. SHA1:  43df5d7
1. CMake configure output: [cmake_configure.txt](https://github.com/trilinos/Trilinos/files/6254883/cmake_configure.txt)

Cc: @alanw0 @jrood-nrel

--- a/packages/seacas/libraries/ioss/src/visualization/cgns/Iovs_cgns_DatabaseIO.h
+++ b/packages/seacas/libraries/ioss/src/visualization/cgns/Iovs_cgns_DatabaseIO.h
@@ -8,7 +8,7 @@
 #define Iovs_cgns_DatabaseIO_h
 
 #include <Ioss_DatabaseIO.h>
-#include <visualization/utils/Iovs_Utils.h>
+#include <Iovs_Utils.h>
 
 namespace Iovs_cgns {
 

--- a/packages/seacas/libraries/ioss/src/visualization/exodus/Iovs_exodus_DatabaseIO.h
+++ b/packages/seacas/libraries/ioss/src/visualization/exodus/Iovs_exodus_DatabaseIO.h
@@ -15,7 +15,7 @@
 #include <Ioss_Field.h>
 #include <Ioss_Map.h>
 #include <Ioss_Utils.h>
-#include <visualization/utils/Iovs_Utils.h>
+#include <Iovs_Utils.h>
 
 #include <algorithm>
 #include <ctime>

